db/items-epoch.lua

```diff
@@ -769,6 +769,16 @@ pfDB["items"]["data-epoch"] = {
       [4129] = 50,
     },
   },
+  [60229] = {
+    ["U"] = { -- dropchance % is a guess
```

Comment:
i think it's safe to say if the drop chance isn't 100% it's always a guess :)

Reply:
Ah yeah, you are right! Should we leave it at 20?